File: src/algo/sort/02.selectionSort.py
def selectionSort(arr):
    for i in range(len(arr) - 1):
        minIndex = i
        for j in range(i+1, len(arr)):
            if(arr[j] < arr[minIndex]):
                minIndex = j

        if i != minIndex:
            arr[i], arr[minIndex] = arr[minIndex], arr[i]

arr = [0,3,20, 9, 18, 7,99]
selectionSort(arr)
print(arr)

File: src/algo/sort/02.selectionSort.js
function selectionSort(arr) {
    var len = arr.length;
    var minIndex, temp;
    for ( var i = 0; i < len -1; i++) {
        minIndex = i;
        for( var j = i +1; j <len; j++) {
            if(arr[j] < arr[minIndex]) {
                minIndex = j;
            }
        }
        
        if(i != minIndex) {
            temp = arr[i];
            arr[i] = arr[minIndex];
            arr[minIndex] = temp;
        }
    }
}

var arr = [0,10,99,37,43,79,60];
selectionSort(arr);
console.log(arr);